lib/cards.js

```diff
@@ -83,26 +81,26 @@ export const CARD_BRAND = {
    * Unavailable
    * @tag Cards
    */
-  UNAVAILABLE: "unavailable"
-}
+  UNAVAILABLE: "unavailable",
+};
 
 /**
  * Card account expiration date
  * @typedef CardExpiration
  * @property {string} month - 2 character month
  * @property {string} year - 2 character year
- * 
+ *
  * @tag Cards
  */
 /**
- * Card account expiration date
- * @typedef CardExternalInfo
- * @property {CardExpiration} expiration - Card expiration date
+ * Card information collected for acquisition.
+ * @typedef LinkCard
```

Comment:
Though I like this name better... I wonder if this will cause some issues for those that are using the type already.  Might be limited.  So, probably OK to let this through.  But something to consider when making these sorts of changes.

Reply:
@eric-moov That type hasn't been released yet, so we're good. Thanks for checking!